### PR TITLE
Adding image assets and analysis tool pages

### PR DESCRIPTION
**Pull request by bergsalex**

### CODE REVIEW

File: docs/analysis-tools/boolean-algebra.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+## Boolean Algebra
+
+The Boolean Algebra Tool performs basic set operations on at least two Gene Sets.
+Results are displayed as lists of genes beloging to one of the three different types of
```
Comment (TessaNM) [inline docs/analysis-tools/boolean-algebra.md:4]: "beloging" --> "belonging"

**Review by C-1337ka — APPROVED** [review]

File: docs/analysis-tools/clustering.md
Comment (TessaNM) [inline docs/analysis-tools/clustering.md:1]: "off" --> "of"

File: docs/analysis-tools/clustering.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+between two clusters is the lowest similarity score between any of the
+genesets in one cluster compared to any of the genesets in the other
+cluster. When two clusters are combined, the genesets within each of the
+clusters are put into a new cluster. No new calculations are needed at
+each iteration because we are simply reusing the similarity scores of
+all the genesets compared to each other.
+
+-   **Algorithm**
+    -   Each gene set is initialized as its own cluster.
+    -   The similarity scores off all the genesets compared to each
```
Comment (TessaNM) [inline docs/analysis-tools/clustering.md:172]: "off" --> "of"

File: docs/analysis-tools/clustering.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+between two clusters is the average similarity score between all of the
+genesets in one cluster compared to all of the genesets in the other
+cluster. When two clusters are combined, the genesets within each of the
+clusters are put into a new cluster. No new calculations are needed at
+each iteration because we are simply reusing the similarity scores of
+all the genesets compared to each other.
+
+-   **Algorithm**
+    -   Each gene set is initialized as its own cluster.
+    -   The similarity scores off all the genesets compared to each
```
Comment (TessaNM) [inline docs/analysis-tools/clustering.md:199]: "off" --> "of"

File: docs/analysis-tools/clustering.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+between two clusters is the highest similarity score between any of the
+genesets in one cluster compared to any of the genesets in the other
+cluster. When two clusters are combined, the genesets within each of the
+clusters are put into a new cluster. No new calculations are needed at
+each iteration because we are simply reusing the similarity scores of
+all the genesets compared to each other.
+
+-   **Algorithm**
+    -   Each gene set is initialized as its own cluster.
+    -   The similarity scores off all the genesets compared to each
```
Comment (TessaNM) [inline docs/analysis-tools/clustering.md:226]: "off" --> "of"

File: docs/analysis-tools/dbscan.md [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+
+![](../assets/images/Circle_Visualization.png "Circle Visualization.png")
+
+### Wires
+
+The other visualization is a wire representation. This shows the
+connections between all genes in the same gene set. The color of each
+gene shows which cluster the gene is in. If a gene is grey, it is
+considered noise. Mousing over a circle will highlight it and show the
+gene ID. By clicking and and holding a gene, you can drag the gene
```
Comment (TessaNM) [inline docs/analysis-tools/dbscan.md:135]: "and" is repeated

File: docs/analysis-tools/dbscan.md [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+### Running the DBSCAN Algorithm
+
+This is the pseudocode for the algorithm.
+
+![DBSCAN
+Pseudocode](../assets/images/DBSCAN_Algorithm_Pseudocode.png "DBSCAN Pseudocode")
+
+Starting in the DBSCAN function, the cluster is first initialized to 0.
+Next, each point is visited only once. For this example, baz will be the
+first gene visited. baz will be first be marked as visited, then the
```
Comment (TessaNM) [inline docs/analysis-tools/dbscan.md:198]: "be first be" --> "be first" or "first be"

File: docs/analysis-tools/dbscan.md [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+entering the expandCluster function, the point P will be added to the
+cluster. The cluster is currently \[baz\]. Next, the algorithm runs
+through all of the neighbors to see if the cluster can be expanded. The
+list of neighbor points is now \[baz, car, Prp31, Arr1\]. First baz is
+looked at, but because it has already been visited, it is not going to
+be checked again. Next, car is checked. Car will then return a list of
+all its neighbors, which are \[car, baz, Prp31, Arr1\]. Then that list
+is checked against the number of minPoints. Since it is greater than or
+equal to minPoints, that list is added to the original list of
+neighbors. So the original neighbors list of \[baz, car, Prp31, Arr1\]
```
Comment (TessaNM) [inline docs/analysis-tools/dbscan.md:226]: Don't need "So"

File: docs/analysis-tools/dbscan.md [hunk 8]
```diff
@@ -0,0 +1,307 @@
...
+that C is a valid cluster. Next, the expandCluster function is called.
+
+The expandCluster will continue to expand the cluster until the edge of
+the cluster is reached. The edge of a cluster is reached when a point
+has a list of neighbors that is less than the number of minPoints. When
+entering the expandCluster function, the point P will be added to the
+cluster. The cluster is currently \[baz\]. Next, the algorithm runs
+through all of the neighbors to see if the cluster can be expanded. The
+list of neighbor points is now \[baz, car, Prp31, Arr1\]. First baz is
+looked at, but because it has already been visited, it is not going to
```
Comment (TessaNM) [inline docs/analysis-tools/dbscan.md:221]: "looked at" could be changed to "checked"

File: docs/analysis-tools/dbscan.md [hunk 9]
```diff
@@ -0,0 +1,307 @@
...
+together. However, the algorithm does not add duplicate genes to the
+list. Therefore, nothing is added to the list and the neighbors list is
+\[baz, car, Prp31, Arr1\]. Then, the gene is added to the current
+cluster if it is not already part of a cluster. car is not a part of any
+other cluster so it is added to the current cluster. Now the cluster
+contains \[baz, car\].
+
+Next, Prp31 is looked at. Its neighbors are \[baz, car, Prp31, Arr1\].
+This list is equal to minPoints, but once again, the list of Prp31's
+neighbors are already in the list of baz's neighbors. So nothing is
```
Comment (TessaNM) [inline docs/analysis-tools/dbscan.md:237]: "So" can be left out

File: docs/analysis-tools/dbscan.md [hunk 10]
```diff
@@ -0,0 +1,307 @@
...
+Now that baz has been checked, it is time to check other genes. Next,
+car is checked. However, it was already visited when handling baz's
+neighbors, so nothing needs to be checked. The same applies for Prp31,
+Arr1, and veli. The next gene to check is Arr2. Arr2's neighbors are
+\[veli, Arr2, CalX\]. This is less than minPoints, so it is marked as
+noise.
+
+![](../assets/images/NewSlide25.jpg "NewSlide25.jpg")
+
+However, just because a gene is marked is noise, does not guarantee it
```
Comment (TessaNM) [inline docs/analysis-tools/dbscan.md:271]: "is" --> "as"

File: docs/analysis-tools/find-variants.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+
+Why Use the Find Variants Tool
+-----------------------------------
+The Find Variants tool traverses a graph database representing the relationships of human or mouse variants and genes. The tool starts with a set of human or mouse genes and outputs a list of variants in the other species using orthologous relationships between genes.
+
+The graph database is built off a fixed and reproducible set of data sourced mostly from Ensembl-104 as well as some other data from JAX, AGR, and GTEx. The data is gathered, processed, and imported into a Neo4J graph database. 
+
+Understanding the Find Variants Tool
+-----------------------------------------
+If the input to the tool is a list of human genes, the tool would find orthologoous genes in mice and variants of those mouse genes either through eQTL relationships or transcript relationships. The tool can also work from mouse genes to human variants.
```
Comment (TessaNM) [inline docs/analysis-tools/find-variants.md:12]: "orthologoous" --> "orthologous"

File: docs/analysis-tools/find-variants.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+
+![](../assets/images/FindVariants_graph.png)
+
+_Figure 1_: Cut of the graph database to map the relationships from a set of human genes to mouse variants.
+
+The tool interacts with the a graph database built using information from AGR and Ensembl for information about genes and their orthologous relationships.
```
Comment (TessaNM) [inline docs/analysis-tools/find-variants.md:18]: "the a" should be "the" or "a"

Comment (TessaNM) [inline docs/analysis-tools/find-variants.md:18]: "for information" --> "for obtaining information"?

File: docs/analysis-tools/geneset-graph.md [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+
+### Using the GeneSet Graph Tool
+
+Access the GeneSet Graph Tool through the [Analyze Genesets](index.md#analyze-gene-sets-tab) tab.
+
+To generate a GeneSet Graph, you must first select gene sets from a project. Projects may be created and updated by uploading GeneSets, searching the GeneWeaver database, or through the use of other tools in the GeneWeaver system. See the documentation for [uploading GeneSets](#uploading-gene-sets), [Search](#searching-geneweaver), or [Manage GeneSets](#gene-set-utilities) to learn more about these functions. To select an entire project or multiple projects for analysis, check the box next to the project name. To select individual GeneSets within a project, click on the **+** beside the project name and check individual GeneSets using the checkboxes. Next, click on the GeneSet Graph icon in the Analysis tools box to the left of the project list. (For users that want to change options, press the green **+** sign before they start the tool).
+
+![](../assets/images/GeneSet_Graph_2.png "fig:GeneSet_Graph_2.png")
+_Figure 2_: GeneSet Graph Selectino Icon.
```
Comment (TessaNM) [inline docs/analysis-tools/geneset-graph.md:24]: "Selectino" --> "Selection"

File: docs/analysis-tools/hisim-graph.md [hunk 14]
```diff
@@ -0,0 +1,198 @@
...
+
+The HiSim Graph Tool is designed to address these goals; it presents a tree of
+hierarchical relationships for a set of input GeneSets. The structure is determined
+solely from the gene overlaps of every combination of GeneSets.
+
+Understanding the Results of the HiSim Graph
+--------------------------------------------
+
+It's best to use the HiSim Graph Tool with knowledge on what set intersections are: If
+GeneSet A contains Gene A, Gene B, and Gene C, and also GeneSet B contains Gene A, Gene
```
Comment (TessaNM) [inline docs/analysis-tools/hisim-graph.md:24]: remove "also"

File: docs/analysis-tools/hisim-graph.md [hunk 15]
```diff
@@ -0,0 +1,198 @@
...
+B, and Gene D. Then the intersection of GeneSet A and GeneSet B will contain Gene A and
```
Comment (TessaNM) [inline docs/analysis-tools/hisim-graph.md:25]: "," instead of "." after "Gene D"

File: docs/analysis-tools/hisim-graph.md [hunk 16]
```diff
@@ -0,0 +1,198 @@
...
+think of it as one of the many measures of accuracy for a map. You want more parsimony,
+but you can't always get full parsimony.
+
+**Gene Aggregation** is a measure of the total node/tree probability. Each node is
+scored based on the intersection of genes and gene sets. Then the product of these
+scores is used to assign an overall tree aggregation probability:
+
+![](../assets/images/Phenome_Map_14.png "fig:Phenome_Map_14.png")
+
+_Figure 12_: Aggregation is another measure of accuracy that balances with parsimony in
```
Comment (TessaNM) [inline docs/analysis-tools/hisim-graph.md:188]: "." after parsimony to start new sentence starting with "In this tool,"

File: docs/analysis-tools/hisim-graph.md [hunk 17]
```diff
@@ -0,0 +1,198 @@
...
+this tool, neither are ever fully accurate alone, but together they are more fine-tuned.
+
+### Permutation Time Limit
+
+The maximum amount of time to spend doing permutations. For example,
+if [Permutations](#permutations) is set to 100,000 and this value is 5 minutes, the
+result with either have 100,000 permutations (if they finished within 5 minutes), or
```
Comment (TessaNM) [inline docs/analysis-tools/hisim-graph.md:195]: "with" --> "will"

File: docs/analysis-tools/jaccard-similarity.md [hunk 18]
```diff
@@ -0,0 +1,87 @@
...
+### PairwiseDeletion
+[Pairwise Deletion](http://www-01.ibm.com/support/docview.wss?uid=swg21475199) is used to pick off problematic missing values from data while still aiming to get the remaining values for comparison-based use:
+
+Values|Obj1|Obj2|Obj3
+:----:|:--:|:--:|:--:
+Length|23|N/A|13
+Width|21|22|14
+Depth|N/A|20|11
+
+_Figure 7_: In Pairwise Deletion, when comparing length, only Obj1 and Obj3 will be compared. When comparing width, all will be compared, and when comparing depth, only Obj2 and Obj3 can be looked at. This prevents missing data from being assigned a default value such as 0 in the system.
```
Comment (TessaNM) [inline docs/analysis-tools/jaccard-similarity.md:87]: "can be looked at" might be better changed to "will be compared"

File: docs/analysis-tools/variant-distance-matrix.md [hunk 19]
```diff
@@ -0,0 +1,40 @@
+**Variant Distance Matrix**
+===============
+About the Variant Distance Matrix Tool
+--------------------------
+The Variant Distance Matrix tool calculates the disimilarity between variant sets in the database and stores the results back into the database. It only calculates the distances between variant sets of the same species and includes sanity checks to ensure that duplicate or improper calculations do not occur. This allows for other tools, most notably the SimilarVariantSet tool, to quickly determine relationships between variant sets. 
+
+The Variant Distance Matrix tool makes one assumption when calculates the distance between variant sets: the genome is static within the same species. In other words, the variant distance matrix tool assumes that any two variant sets for which the distance is calculated for have the same reference genome. This allows the tool to assume that the distance between two variant sets is only a function of the differences between the variants in the variant sets and not any other outside variables. Since both variant sets have the same static genome, the distance is calculated using the following formula:
```
Comment (TessaNM) [inline docs/analysis-tools/variant-distance-matrix.md:7]: "calculates" --> "calculating"

Comment (TessaNM) [inline docs/analysis-tools/variant-distance-matrix.md:7]: remove "for" from "which the distance is calculated for have the same reference genome."

File: docs/analysis-tools/variant-distance-matrix.md [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+
+![variant distance equation](images/variant_distance_equation.png)
+
+Using the Variant Distance Matrix Tool
+--------------------------
+This tool mainly functions as a helper tool to the VariantBatchUpload and SimilarVariantSet tools and is unable to be called from the website. Upon the upload of a VariantSet, multiple Variant Distance Matrix instances are forked off to calculate the distances between the new VariantSet and all existing variant sets in the database. The SimilarVariantSet uses this tool when any two uploaded variant sets do not already have an existing distance in the database. The SimilarVariantSet then forks of instances of the Variant Distance Matrix tool in order to calculate these distances.
```
Comment (TessaNM) [inline docs/analysis-tools/variant-distance-matrix.md:13]: I'm not sure what this means: "The SimilarVariantSet then forks of instances of" in the last sentance

File: docs/analysis-tools/variant-distance-matrix.md [hunk 21]
```diff
@@ -0,0 +1,40 @@
...
+-------
+There is one option that can be used in conjunction with this tool.
+
+### Disable Sanity Check
+ The Disable Sanity Check option can be set to True in order to speed up distance matrix creation. This option disables the two sanity checks that are present in the tool does the following checks.
+
+  * the genesets are of the same species
+  * the distance hasn't already been calculated
+  * the geneset is a variantset
+  If it is assured that the genesets passed in fufill these criteria, the sanity checks can be turned off in order to speed up the creation of the distance matrix.
```
Comment (TessaNM) [inline docs/analysis-tools/variant-distance-matrix.md:29]: "fufill" --> "fulfill"

File: docs/analysis-tools/variant-pipeline.md [hunk 22]
```diff
@@ -0,0 +1,50 @@
+## Variant Pipeline
+
+5 api endpoints are created for the following 5 pipelines: eggo, eggv, eggr, eggix, and eggin. Each pipeline has a get endpoint to retrieve the information from running each entire pipeline. 
+ 
+The pipeline can be found on Swagger UI through running Geneweaver on Watson and use this link: http://localhost:8889/api/
```
Comment (TessaNM) [inline docs/analysis-tools/variant-pipeline.md:5]: "use" --> "using"

**Review by TessaNM — APPROVED** [review]

**Pull request merged**